WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt

```diff
@@ -109,10 +110,11 @@ class PostListFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        val onlyUser = requireNotNull(arguments).getBoolean(EXTRA_POST_LIST_ONLY_USER)
         val postListType = requireNotNull(arguments).getSerializable(EXTRA_POST_LIST_TYPE) as PostListType
         viewModel = ViewModelProviders.of(this, viewModelFactory)
                 .get<PostListViewModel>(PostListViewModel::class.java)
-        viewModel.start(site, postListType)
+        viewModel.start(site, onlyUser, postListType)
```

Comment:
I _think_ we'll always use `postListType` and the author filter together. So, how about we introduce a new class to hold both the author filter and the list type together? We might even be able to use the `PostListDescriptor` for this, because that's pretty much what we'll convert these values to.

What do you think @malinajirka @drspaceboo ?

Reply:
I think with the current implementation the location where we could combine these is in the `PostsPagerAdapter`. That class is currently very simple with no business logic in it.

After that point in the code the next time they are used together is in the `start` method of `PostListViewModel`. My thought would be to keep the creation (and choice between REST or XMLRPC) in the ViewModel.

However you both have more experience with this pattern, so will defer that decision to you two.

Reply:
> My thought would be to keep the creation (and choice between REST or XMLRPC) in the ViewModel.

We should definitely do that and good point on the REST/XMLRPC descriptors. We can probably introduce a very simple class to tie them up and use that instead of individually setting the status. We could maybe do that within FluxC's descriptor instead, so it's always used together. What do you think @drspaceboo?

Reply:
@oguzkocer The one place they will be used together is in that start function, where they are turned into a `PostListDescriptor`. I think that makes sense currently. Perhaps that may change in a later PR if we implement the Fragments themselves being able to update how they are filtering.

My vote would be to leave it as it is and add that in if we need it following YAGNI.

Reply:
Sounds good to me 👍